packages/aws-rfdk/lib/deadline/lib/thinkbox-docker-images.ts

```diff
@@ -33,23 +33,32 @@ import {
 } from '.';
 
 /**
- * Choices for signifying the user's stance on the terms of the AWS Thinkbox End-User License Agreement (EULA).
- * See: https://www.awsthinkbox.com/end-user-license-agreement
+ * Choices for signifying the user's stance on the terms of the AWS Customer Agreement and IP License.
+ *
+ * See:
+ * * AWS Customer Agreement: https://aws.amazon.com/agreement/
+ * * AWS Intellectual Property License: https://aws.amazon.com/legal/aws-ip-license-terms/
  */
-export enum AwsThinkboxEulaAcceptance {
+export enum AwsCustomerAgreementAndIpLicenseAcceptance {
   /**
-   * The user signifies their explicit rejection of the tems of the AWS Thinkbox EULA.
+   * The user signifies their explicit rejection of the tems of either the AWS Customer Agreement and/or
+   * AWS Intellectual Property License.
    *
-   * See: https://www.awsthinkbox.com/end-user-license-agreement
+   * See:
+   * * AWS Customer Agreement: https://aws.amazon.com/agreement/
+   * * AWS Intellectual Property License: https://aws.amazon.com/legal/aws-ip-license-terms/
    */
-  USER_REJECTS_AWS_THINKBOX_EULA = 0,
+  USER_REJECTS_AWS_CUSTOMER_AGREEMENT_AND_IP_LICENSE = 0,
```

Comment:
Why we need so long name here? I believe since enum is `AwsCustomerAgreementAndIpLicenseAcceptance` and cannot be used without enum name it could be just `USER_REJECTS` and `USER_ACCEPTS`. What do you think?

Reply:
Yeah, the long name isn't great, but the idea is to be as explicit as possible so there's no chance that the end-user doesn't understand exactly what they are agreeing to.

Reply:
I think there is no chance that user will type this name instead of copy it. But I have no idea what way is the most proper. Just suggestion.